devices/SPS30/Entities/VersionInformation.cs

```diff
@@ -0,0 +1,68 @@
+﻿//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace Iot.Device.SPS30.Entities
+{
+    /// <summary>
+    /// Parsed response after requesting version information.
+    /// </summary>
+    public class VersionInformation
+    {
+        /// <summary>
+        /// Parsed response after requesting version information.
+        /// </summary>
+        /// <param name="data">Raw data from the response</param>
+        /// <exception cref="ArgumentOutOfRangeException">When less than 7 bytes are provided</exception>
+        public VersionInformation(byte[] data)
+        {
+            if (data.Length < 7)
+                throw new ArgumentOutOfRangeException(nameof(data), "Unexpected array size. Expecting at least 7 bytes.");
+            FirmwareMajorVersion = data[0];
+            FirmwareMinorVersion = data[1];
+            // data[2] is reserved
+            HardwareRevision = data[3];
+            // data[4] is reserved
+            SHDLCProtocolMajorVersion = data[5];
+            SHDLCProtocolMinorVersion = data[6];
+        }
+
+        /// <summary>
+        /// Firmware major version
+        /// </summary>
+        public int FirmwareMajorVersion { get; private set; }
+
+        /// <summary>
+        /// Firmware minor version
+        /// </summary>
+        public int FirmwareMinorVersion { get; private set; }
+
+        /// <summary>
+        /// Hardware rivision
+        /// </summary>
+        public int HardwareRevision { get; private set; }
+
+        /// <summary>
+        /// SHDLC protocol major version
+        /// </summary>
+        public int SHDLCProtocolMajorVersion { get; private set; }
```

Comment:
Same remark for the abbreviation, should be `ShdlcProtocol`

And then: `public Version ShdlcProtocolVersion { get; private set; }`